executor/poolmgr/gpm.go

```diff
@@ -178,3 +178,13 @@ func (gpm *GenericPoolManager) eagerPoolCreator() {
 		gpm.CleanupPools(envs.Items)
 	}
 }
+
+func (gpm *GenericPoolManager) getEnvPoolsize(env *crd.Environment) int32 {
```

Comment:
Just curious, why does this return int32 and not int?

Reply:
This is because the `Replicas` in `DeploymentSpec` needs a int32

Reply:
@soamvasani I will merge this if looks good to you?